apis/meta/conditions.go

```diff
@@ -75,48 +49,16 @@ const (
 	SuspendedReason string = "Suspended"
 )
 
-// DeepCopyInto is a deepcopy function, copying the receiver, writing into out. in must be non-nil.
-func (in *Condition) DeepCopyInto(out *Condition) {
-	*out = *in
-	in.LastTransitionTime.DeepCopyInto(&out.LastTransitionTime)
-}
-
-// DeepCopy is a deepcopy function, copying the receiver, creating a new Condition.
-func (in *Condition) DeepCopy() *Condition {
-	if in == nil {
-		return nil
-	}
-	out := new(Condition)
-	in.DeepCopyInto(out)
-	return out
-}
-
-// GetCondition returns the Condition from the given slice that matches the
-// given type.
-func GetCondition(conditions []Condition, conditionType string) *Condition {
```

Comment:
Uncertain if to keep this and refactor so it's just a wrapper for [FindStatusCondition](https://github.com/kubernetes/apimachinery/blob/v0.19.3/pkg/api/meta/conditions.go#L73) or just ditch it (as is done now) and switch to that one directly where applicable.

Reply:
I think this is fine, we should be using FindStatusCondition.